ArduPlane/quadplane.cpp

```diff
@@ -2195,6 +2192,17 @@ void QuadPlane::vtol_position_controller(void)
         poscontrol.last_run_ms = now_ms;
     }
 
+    // avoid running the z controller in approach and airbrake if we're not already running it
+    // and tilt is more than tilt max
+    bool suppress_z_controller = false;
+    if (tiltrotor.enabled() && poscontrol.get_state() == QPOS_AIRBRAKE) {
+        if ((now_ms - last_pidz_active_ms > 2000 && tiltrotor.tilt_over_max_angle()) ||
+            tiltrotor.current_tilt >= tiltrotor.get_fully_forward_tilt()) {
+            suppress_z_controller = true;
+            hold_stabilize(0.01);
```

Comment:
how about a force air mode bool, would make it a little clearer, and also should be 1% better. 
```suggestion
            hold_stabilize(0.0, true);
```

Reply:
as we discussed, lower throttle isn't necessarily better, we may get more drag from a bit more. I suspect we'll end up needing a parameter for throttle in airbrake eventually